docs/specification.md

```diff
@@ -587,87 +596,47 @@ Parameter (in Michelson)
 
 - Returns token pause status.
 
-**Issuing and destroying tokens**
---------------------------
+### Issuing and destroying tokens
 
 **configure_minter**
 
 Types
 ```
-configure_minter_param =
-  ( address :sender
-  , address :minter
+configure_minter =
+  ( address :minter
+  , nat     :current_minting_allowance
+  , nat     :new_minting_allowance
   )
-
-configure_minter = configure_minter_param
 ```
 
 Parameter (in Michelson)
 ```
 (pair %configure_minter
-  (address %sender)
   (address %minter)
+  (pair (nat %current_minting_allowance) (nat %new_minting_allowance))
 )
 ```
 
-- Adds `minter` to global minter list to allow him mint tokens.
+- Adds `minter` to the global minter list to allow him to mint tokens (if `minter` is not in the list already).
 
-- Sender must be master minter.
+- Sets the specified minting allowance (`new_minting_allowance`) for this minter.
 
-- Minter cannot mint tokens untill `set_minting_allowance`
-  is specified.
+- `current_minting_allowance` must be explicitly passed.
+If it does not match the actual minting allowance, the transaction MUST fail.
```

Comment:
> If it does not match the actual minting allowance

Or if the minter does not exist already, right?

Reply:
I think it's simpler to not distinguish removed minters and minters with 0 minting allowance. I don't see how it can be useful to distinguish them. But I'll add it to the list of questions just to be sure.

Reply:
Oh wait, there is important difference: a minter with 0 minting allowance can burn tokens and removed minter can't :thinking:

Reply:
Okay, then I guess it's cleaner to change `current_minting_allowance` to `option nat` and treat `None` as "not a minter".

Reply:
Ok, so does that mean that we can ditch the global minter list now, because it seems to be redundant?

Reply:
Well, "global minter list" is now present in only two places: `configure_minter` and `remove_minter`.

The CENTRE doc also says "the list of minters":
> This will remove the minter from the list of minters…

I guess I'll just remove the word "global" because it is indeed redundant now (there are no "local" lists of minters).

Reply:
Conceptually we should maintain a list of minters. We don't have to store them literally in a `list`, in practice we will probably store them in a `map` or a `big_map`. But from end user's perspective there is still a list.